include/bm/bm_sim/actions.h

```diff
@@ -751,6 +768,8 @@ class ActionFn :  public NamedP4Object {
   // signal the end.
   void parameter_start_vector();
   void parameter_end_vector();
+  void start_field_list();
+  void end_field_list();
```

Comment:
please rename these to `parameter_start_field_list` and `parameter_end_field_list` for consistency